layers/poky/patch/0001-Fix-automated-runtime-testing-using-SystemdbootTarge.patch

```diff
@@ -0,0 +1,69 @@
+From 7f2988cc0e50062277908b485c3a623d025e4cf9 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Erik=20Bot=C3=B6?= <erik.boto@pelagicore.com>
+Date: Thu, 18 May 2017 15:12:56 +0200
+Subject: [PATCH] Fix automated runtime testing using SystemdbootTarget
+
+---
+ meta/classes/testimage.bbclass           | 2 +-
+ meta/lib/oeqa/controllers/masterimage.py | 4 ++--
+ meta/lib/oeqa/utils/sshcontrol.py        | 9 +++------
+ 3 files changed, 6 insertions(+), 9 deletions(-)
+
+diff --git a/meta/classes/testimage.bbclass b/meta/classes/testimage.bbclass
+index 6b6781d860..3994d18553 100644
+--- a/meta/classes/testimage.bbclass
++++ b/meta/classes/testimage.bbclass
+@@ -158,7 +158,7 @@ def testimage_main(d):
+         bootparams = None
+         if d.getVar('VIRTUAL-RUNTIME_init_manager', '') == 'systemd':
+             bootparams = 'systemd.log_level=debug systemd.log_target=console'
+-        target.start(extra_bootparams=bootparams)
++        target.start(params=bootparams)
+         starttime = time.time()
+         result = tc.runTests()
+         stoptime = time.time()
+diff --git a/meta/lib/oeqa/controllers/masterimage.py b/meta/lib/oeqa/controllers/masterimage.py
+index 9ce3bf803d..4981d3d425 100644
+--- a/meta/lib/oeqa/controllers/masterimage.py
++++ b/meta/lib/oeqa/controllers/masterimage.py
+@@ -108,7 +108,7 @@ class MasterImageHardwareTarget(oeqa.targetcontrol.BaseTarget, metaclass=ABCMeta
+             time.sleep(10)
+             self.power_ctl("cycle")
+         else:
+-            status, output = conn.run("reboot")
++            status, output = conn.run("{ sleep 1; reboot; } > /dev/null &")
```

Comment:
Actually no, this change doesn't seem to break it but it doesn't help anything either.